src/effects/DynamicRangeProcessorHistoryPanel.cpp

```diff
@@ -140,27 +144,38 @@ void DrawHistory(
          targetPoints.emplace_back(x, yt);
       }
 
-      gc.SetPen(wxPen { theTheme.Colour(clrResponseLines), followLineWidth });
-      if (segment.size() == 1)
+      // Draw the follow line in black, solid line ...
+      gc->SetPen(wxPen { wxColour { 0, 0, 0 } });
```

Comment:
wxWidgets defines set of pen/brush constants. 
Could be replaced with `*wxBLACK_PEN`